lib/verk/queue_stats.ex

```diff
@@ -10,39 +10,36 @@ defmodule Verk.QueueStats do
   { queue_name, running_jobs_counter, finished_jobs_counter, failed_jobs_counter }
   """
   use GenEvent
+  alias Verk.QueueStatsCounters
 
-  @table :queue_stats
-
-  @doc """
-  Lists the queues and their stats
-  """
-  def all do
```

Comment:
I had to remove this function. It didn't make much sense to keep it as now we are just keeping the increment we should send to Redis.